packages/layout/src/components/helplink/helplink.tsx

```diff
@@ -2,28 +2,55 @@ import React from 'react';
 import Collapsible from 'react-collapsible';
 import { Hint } from '../hint/hint';
 import { ArrowDown } from '@tpr/icons';
-import { Flex } from '@tpr/core';
 import Styles from './helplink.module.scss';
 
 type HelpLinkProps = {
 	title: string;
-	content: string;
```

Comment:
This change will break existing apps where the `Helplink` component is used and the `content` prop is received.
If we keep `content` here (along with the new props) and then simply pass it to the component, instead of children, will it work?